services/slam/builtin/builtin.go

```diff
@@ -343,11 +343,23 @@ func configureCameras(ctx context.Context, svcConfig *AttrConfig, deps registry.
 			if !ok {
 				return "", nil, transform.NewNoIntrinsicsError("Intrinsics do not exist")
 			}
+
 			err = intrinsics.CheckValid()
 			if err != nil {
 				return "", nil, err
 			}
+
+			props, err := cam.Properties(ctx)
+			if err != nil {
+				return "", nil, errors.Wrap(err, "error getting camera properties for slam service")
+			}
+
+			brownConrady, ok := props.DistortionParams.(*transform.BrownConrady)
```

Comment:
NOTE:

It turns out that `props.DistortionParams` always is either a struct with 5 members, each containing a float (if the config input `"distortion_parameters": {}` is provided then `props.DistortionParams` is `&{0 0 0 0 0}`.

If `"distortion_parameters"` in the config is any object value (for example: `"distortion_parameters": {"mickey": {"mouse": true}}` it will always succeed in creating that five element struct, ignoring the fiends which are not members of `BrownConrady{}` and setting any unset members to zero values. 

Setting `"distortion_parameters": null` results in `props.DistortionParams` being `nil`, which results in `brownConrady` being `nil` and (counterintuitively for a go noob like me) `ok` being `true`. Eric explained why this occurs in this [ticket](https://viam.atlassian.net/browse/RSDK-1660) (asserting nil to a pointer type can succeed even if the resulting value will be `nil`) & links to this faq: https://go.dev/doc/faq#nil_error.
 
Setting `"distortion_parameters"`  to any type other than `null` or `object` results in rdk crashing as soon as it gets the config.

B/c of rdk parses the config upstream & how go handles zero values, I believe (but would love to be proven wrong) I'm unable to distinguish between a malformed `"distortion_parameters"` config. Given that we will likely want to be able to handle different distortion params in the future or refactor them, as Bijan mentions [here](https://viam.atlassian.net/browse/RSDK-1271?focusedCommentId=13552) , I figured treating the case of either the type assertion failing or the type assertion succeeding but resulting in a `nil` pointer, as the same case, given that I don't know how the behavior of this type assertion, as currently written, will change if / when the possible valid structures of  `"distortion_parameters"` changes.